package.json

```diff
@@ -61,7 +61,7 @@
         "eslint-plugin-mocha": "10.1.0",
         "mocha": "10.2.0",
         "supertest": "6.3.3",
-        "typescript": "^5.0.4"
+        "typescript": "^5.2.2"
```

Comment:
@adam4leos 
Seems like we ought to keep the `typescript` version the same for `dwn-server` as it is for `dwn-sdk-js` and all of the `web5-js` packages: 5.1.6:
https://github.com/TBD54566975/dwn-sdk-js/blob/6c381a2e72f1d94b61c563daa313c83936c31640/package.json#L137

and bump them together